### ISSUE TITLE
User

### ISSUE THREAD

**Issue by Krishna8077078** [ConnectyCube/connectycube-reactnative-samples#173]
How to get user Without Pagination From Api...

**Comment by ccvlad** [comment]
Hi, @Krishna8077078 

use the code below:
```javascript
ConnectyCube.users.get(userId).then(response => console.log(response.user));
ConnectyCube.users.get({ login: 'some_login' }).then(response => console.log(response.user));
ConnectyCube.users.get({ email: 'some_email@gmail.com' }).then(response => console.log(response.user));
```

**Comment by Krishna8077078** [comment]
I think u can't get it what i am trying to say...means is--how to retrieve users from API as without pagination..and is there any other way to filter use via a phonebook contact list.

**Comment by ccvlad** [comment]
You will receive response with one user w/o pagination if you would use the code I have provided before (see full possible parameters in our [docs](https://developers.connectycube.com/js/authentication-and-users?id=retrieve-users)).
You also are able to get users by [filters](https://developers.connectycube.com/server/users?id=retrieve-all-users) - `ConnectyCube.users.get(filter)`.
Check the Adressbook API to get phone's contacts - https://developers.connectycube.com/server/users?id=retrieve-all-users

I did not understand your case. What is wrong with the pagination?
